pkg/webhooks/clusterqueue_webhook_test.go

```diff
@@ -389,6 +402,16 @@ func TestValidateClusterQueueUpdate(t *testing.T) {
 			oldClusterQueue: testingutil.MakeClusterQueue("cluster-queue").QueueingStrategy("BestEffortFIFO").Obj(),
 			wantErr:         nil,
 		},
+		{
+			name: "existing cluster queue created with older Kueue version that has a nil borrowWithinCohort field",
+			newClusterQueue: testingutil.MakeClusterQueue("cluster-queue").
+				Preemption(kueue.ClusterQueuePreemption{ReclaimWithinCohort: kueue.PreemptionPolicyNever}).
+				Obj(),
+			oldClusterQueue: testingutil.MakeClusterQueue("cluster-queue").
+				Preemption(kueue.ClusterQueuePreemption{ReclaimWithinCohort: kueue.PreemptionPolicyNever}).
+				Obj(),
+			wantErr: nil,
+		},
```

Comment:
I think that this case is duplicated with `TestValidateClusterQueue`'s one since the `ValidateClusterQueueUpdate` calls the `ValidateClusterQueue`, then the `ValidateClusterQueue` calls `validatePreemption`.

So, only the case in the `TestValidateClusterQueue` should be enough.

Reply:
You're right. I've removed it.